### ISSUE TITLE
Misturando idiomas & Contextualização do Jogo

### ISSUE THREAD

**Issue by NixieCat** [Treeofsavior/PortugueseTranslation#123]
Misturando idiomas: Eu vi que algumas pessoas estavam querendo manter alguns termos como Quest e Loot no jogo. Esse termos são usadas pelos jogadores, mas não tem sentido fazer um tradução e manter palavras de um outro idioma quando existe uma tradução. A exceção seria termos que não possuem traduções ou que o contexto peça. 

Exemplo de exceções. Um clérigo usando termos em Latim. O nome da classe intraduzíveis ou que perderiam o significado ao ser traduzido como  Wugushi, Shinobi, Shadu e por aí vai...

Contextualização do Jogo: Na hora de traduzir, temos que considerar a contextualização do jogo. Aparentemente o jogo se passa na Idade Média, então eu acho interessante usarmos termos que remetem algo mais medieval. Mas sem exageros é claro, ninguém quer utilizar um dicionário para jogar.

**Comment by yalmeida69** [comment]
Concordo com traduzir tudo que estiver EM INGLÊS e possuir uma tradução factível. Os termos em outras línguas que trazem os traços de individualidade pra lore do jogo acho desnecessário mexer.

**Comment by evtutchenko** [comment]
O problema é que "quest" não tem uma tradução boa para português (boa = não passa completamente a ideia original), e os termos que poderiam ser utilizados já serão para outras traduções (como já foi dito sobre o sistema de missões do jogo). Loot pode ser traduzido, mas em minha opinião ela já há um tempo se tornou um jargão bastante conhecido no meio dos jogos.

**Comment by billarkwright** [comment]
Pode-se muito bem traduzir quest para missões, logo que boa parte do jogo você estará ligado a elas. E o sistema de missões do jogo pode muito bem encontrar uma alternativa para o mesmo, como já foi sugerido em outro issue.

**Comment by NixieCat** [comment]
@evtutchenko Sim, eu concordo que o loot já é um termo bastante difundido, mas eu acho que a tradução não deveria ser levada em conta os jogadores. Porque se fosse o caso, personagem teria que ser chamado de Char, já que é mais difundido que personagem.

Na minha opnião há palavras que se encaixam bem em quest, como missão, jornada, expedição, busca.

**Comment by lmmfranco** [comment]
Talvez fosse interessante se inspirar em outras adaptações de MMO's para português, como por exemplo World of Warcraft da Blizzard.

**Comment by AngelClaw** [comment]
Temos que levar em conta que muitos jogadores podem não conhecer o termo, por maais difundido que seja. Muitos jogadores irão preferir jogar em inglês, mas quem não quiser precisa de uma tradução precisa e respeitosa.

**Comment by evtutchenko** [comment]
@NixieCat @AngelClaw  Eu não digo que é para deixar apenas por que certa palavra já é tão usada no meio que todos sabem o que significa. Não é exatamente isso. O problema que eu enxergo é que traduzir algo sistêmico para o português, sendo a palavra escolhida obscura no vocabulário comum, não chega ao objetivo de deixar as coisas mais claras. @NixieCat você também levou ao extremo a minha ideia. Personagem não é uma palavra obscura na língua portuguesa.

**Comment by yalmeida69** [comment]
Eu acho que não precisa ser uma tradução ao pé da letra (como "espólio" que eu tinha sugerido pra "loot"), mas algo que passe o significado já adianta (seguindo o mesmo exemplo de "loot", até um simples "itens" já atenderia). E ainda temos outras opções, como inverter as traduções - seria o caso de traduzir "quest" pra "missão" (já que "tarefa" não passa o mesmo senso de importância), e traduzir as "missions" pra algo diferente, dependendo do contexto.

Só não vamos esquecer que por mais que essa tradução em inglês seja a nossa base e ponto de partida, ela não necessariamente é a verdade absoluta. Ela também foi feita pela comunidade e está tão propensa a falhas quanto a nossa. Às vezes pode ser interessente utilizarmos nossa experiência com o português e com RPGs ou coisas do tipo e escolhermos um termo mais adequado do que o equivalente em inglês, pelo bem da compreensão.

**Comment by AngelClaw** [comment]
Não nos esqueçamos que também temos neologismos como ferramenta, que são cada dia mais presentes na nossa língua, especialmente com a disseminação do inglês como língua "universal", entre aspas pq odeio esse termo.
